sdk/eventhub/event-hubs/src/impl/partitionGate.ts

```diff
@@ -43,16 +41,4 @@ export class PartitionGate {
   remove(partitionId: string | "all") {
     this._partitions.delete(partitionId);
   }
-
-  private _validatePartitionId(partitionId: string) {
```

Comment:
Are we doing this because we have higher level validation somewhere else? Or are we just getting out of the "validating the type of this value" altogether?

Reply:
At a higher level, we ensure that partitionId passed down is always a string.

The client should not make the assumption that partitionIds are of the form 0,1,2,3 because there is no such contract with the service. As far as the service is concerned, partitionIds are strings.